### ISSUE TITLE
Can FLAME estimate eyeball rotation when only using landmarks for fitting?

### ISSUE THREAD

**Issue by hengfei-wang** [Rubikplayer/flame-fitting#33]
Flame has the parameters which can control eyeball rotation. But I saw that the landmark used for flame fitting doesn't include pupil center which may make it impossible to estimate eyeball rotation parameters only from landmarks?

**Comment by TimoBolkart** [comment]
You are right, estimating eye gaze by optimizing FLAME's eyeball pose parameters requires additional landmarks for the pupil center. If you are using a landmark predictor that provides pupil center locations, one can easily add the distance between FLAME's projected pupil center and the predicted pupil locations (i.e., same as the existing landmark loss but for an additional pair of points) as additional loss and optimize for FLAME's eyeball parameters.

**Comment by hengfei-wang** [comment]
Thank you!  I will give it a try.

**Comment by ArjunZ786** [comment]
I would like to do something similar not for eyes but like add a few more known landmarks. Were you able to add it? Is so can you share steps or code to do the same?
